querydsl-spatial/src/main/java/com/querydsl/spatial/jts/JTSPolygonExpression.java

```diff
@@ -40,7 +40,7 @@
     private volatile NumberExpression<Integer> numInteriorRing;
 
     @Nullable
-    private volatile JTSLineStringExpression<LineString> exterorRing;
+    private volatile JTSLineStringExpression<LineString> exteriorRing;
```

Comment:
Good catch. So I should just leave a comment in the code not to change that misspelling when I backport this branch?